js/modules/sankey.src.js

```diff
@@ -469,11 +469,17 @@ seriesType('sankey', 'column',
                         // Hanging layout for organization chart
                         if (fromNode.options.layout === 'hanging') {
                             node.hangsFrom = fromNode;
-                            node.column += fromNode.linksFrom.findIndex(
-                                function (link) {
-                                    return link.toNode === node;
+                            i = -1; // Reuse existing variable i
+                            fromNode.linksFrom.find(
```

Comment:
Unfortunately `Array.prototype.find` is also [not available in IE11](https://caniuse.com/#feat=array-find) :(

The fastest code would be a good old `for` loop with a `break`, so we don't have to continue iterating once the item is found. Or a `while`.

Reply:
> Unfortunately Array.prototype.find is also not available in IE11 :(

Nooo, I did not think of that. Haha, this was ironic 😆 

> The fastest code would be a good old for loop with a break, so we don't have to continue iterating once the item is found. Or a while.

`find` stops iterating once it matches a value. Could I replace `Array.prototype.find` with `Highcharts.find`, or is still a `for` loop preferred?

Reply:
Ok, yes of course `find` is okay.

Reply:
I updated the branch with this change now, just waiting for CI to make final confirmation.